arch/riscv/Kconfig.isa

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2022 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+menu "RISCV ISA and extensions"
+
+choice
+	prompt "Core ISA"
+	default RISCV_ISA_RV64I if 64BIT
+	default RISCV_ISA_RV32I
+
+config RISCV_ISA_RV32I
+	bool "RV32I Base Integer Instruction Set - 32bit"
+
+config RISCV_ISA_RV32E
+	bool "RV32E Base Integer Instruction Set (Embedded) - 32bit"
+
+config RISCV_ISA_RV64I
+	bool "RV64I Base Integer Instruction Set - 64bit"
+
+config RISCV_ISA_RV128I
+	bool "RV128I Base Integer Instruction Set - 128bit"
+
+endchoice
+
+config RISCV_ISA_EXT_M
+	bool "(M) - Standard Extension for Integer Multiplication and Division"
+	default y
```

Comment:
The extension Kconfigs should not have defaults since they are meant to be selected by the SoC Kconfig, and SoC Kconfig in itself does not have a way of de-selecting these (you CAN do it in defconfig, but then you will have the extension lists defined at two different places, which is not ideal).

These extension Kconfigs should be verbosely selected by the SoC Kconfigs.

Reply:
yeah, this bothered me as well, it was defaulted already to `y` so I ported this over.